configure

```diff
@@ -516,8 +517,11 @@ elif system == 'darwin':
 
 if 'ARCH' in os.environ.keys():
   march = os.environ['ARCH']
-  report ('Machine architecture set by ARCH environment variable to: ' + march + '\n')
-  cpp_flags += [ '-march='+march ]
+  if march:
```

Comment:
OK, seems fine [according to the python docs](https://docs.python.org/3/library/stdtypes.html#truth-value-testing). Let's merge.